### PR TITLE
resolves RTL issue ABU 1136 (Bulletpoint 2)

### PR DESCRIPTION
**Pull request by chetan-hajare**

### CODE REVIEW

**Comment by lc-thomasberger** [comment]
Looking good!
For some reason the animation of the scaleindicator is laggy ... which should not be a showstopper.

File: less/slider.less [hunk 1]
```diff
@@ -138,6 +147,9 @@
         }
         .rangeslider {
           background-color: @background-color;
+          .dir-rtl & {
+            transform: rotateY(180deg);
```
Comment (lc-thomasberger) [inline less/slider.less]: this breaks in ie 8/9
http://caniuse.com/#feat=transforms2d

**Comment by chetan-hajare** [comment]
Thanks! for suggestion. I will look into it.

**Comment by lc-thomasberger** [comment]
[this](http://stackoverflow.com/questions/4617220/css-rotate-property-in-ie) might be useful for ie8 support

**Pull request merged**